### ISSUE TITLE
Error: Cannot find module 'preview-email', in version 2.0.0

### ISSUE THREAD

**Issue by Juraj-Masiar** [nest-modules/mailer#1176]
**Describe the bug**
I've updated to `"@nestjs-modules/mailer": "^2.0.0",` and now the project fails to run, throwing following error:
```
[15:54:12] Starting compilation in watch mode...

[15:54:17] Found 0 errors. Watching for file changes.

Error: Cannot find module 'preview-email'
Require stack:
- C:\Users\juraj\git\gsd_server\node_modules\@nestjs-modules\mailer\dist\mailer.service.js
- C:\Users\juraj\git\gsd_server\node_modules\@nestjs-modules\mailer\dist\mailer-core.module.js
- C:\Users\juraj\git\gsd_server\node_modules\@nestjs-modules\mailer\dist\mailer.module.js
- C:\Users\juraj\git\gsd_server\node_modules\@nestjs-modules\mailer\dist\index.js
- C:\Users\juraj\git\gsd_server\node_modules\@nestjs-modules\mailer\index.js
- C:\Users\juraj\git\gsd_server\dist\src\email\email.service.js
- C:\Users\juraj\git\gsd_server\dist\src\auth\auth.service.js
- C:\Users\juraj\git\gsd_server\dist\src\auth\auth.controller.js
- C:\Users\juraj\git\gsd_server\dist\src\auth\auth.module.js
- C:\Users\juraj\git\gsd_server\dist\src\sync-dials\sync-dials.module.js
- C:\Users\juraj\git\gsd_server\dist\src\app.module.js
- C:\Users\juraj\git\gsd_server\dist\src\main.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:1143:15)
    at Function.Module._load (node:internal/modules/cjs/loader:984:27)
    at Module.require (node:internal/modules/cjs/loader:1231:19)
    at require (node:internal/modules/helpers:179:18)
    at Object.<anonymous> (C:\Users\juraj\git\gsd_server\node_modules\@nestjs-modules\mailer\dist\mailer.service.js:8:22)
    at Module._compile (node:internal/modules/cjs/loader:1369:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1427:10)
    at Module.load (node:internal/modules/cjs/loader:1206:32)
    at Function.Module._load (node:internal/modules/cjs/loader:1022:12)
    at Module.require (node:internal/modules/cjs/loader:1231:19)
```

All of my NestJS libraries are updated to latest packages (10.x.x).

**Comment by joaodematejr** [comment]
https://www.npmjs.com/package/preview-email

**Comment by Juraj-Masiar** [comment]
Oh, right, I need to install that manually :), thanks!

**Comment by mgrunberg** [comment]
I'm facing the same problem. 
I don't understand why I must install a devDependency in my project.
I mean, version 2.0.0 moved `preview-email` from `dependencies` to `devDependencies` (package.json). Could that be a bug?

**Comment by mgrunberg** [comment]
In my case, the problem got solved in version 2.0.2.  `preview-email` is optional, I was able to use that version without installing `preview-email`
